apps/web/core/components/settings/task-statuses-form.tsx

```
@@ -87,43 +87,45 @@ export const TaskStatusesForm = ({ formOnly = false, onCreated }: StatusForm) =>
-					icon: values.icon
-				})?.then(() => {
-					setEdit(null);
-					refetch();
-				});
+				console.log('[WEB][TaskStatusesForm] Creating new status with data:', requestData);
+
+				try {
+					await createTaskStatus(requestData)?.then(() => {
+						console.log('[WEB][TaskStatusesForm] Status created successfully');
+						!formOnly && setCreateNew(false);
+						onCreated && onCreated();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)